.github/workflows/dependency-tree-update.yml

```diff
@@ -88,7 +88,7 @@ jobs:
         uses: tradeshift/create-pull-request@v3
         with:
           commit-message: 'chore: update dependency tree for ${{ steps.formatPath.outputs.path }}/package.json'
-          token: ${{ github.token }}
+          token: token: ${{ secrets.github-token }}
```

Comment:
one too many tokens in here :see_no_evil:

Reply:
fixed 🤠 🤡